edge-modules/MetricsCollector/Settings.cs

```diff
@@ -59,13 +43,28 @@ public class Settings
             this.UploadTarget = uploadTarget;
```

Comment:
There are 2 values of upload target enum - eventhub and log analytics; what if upload target is eventhub? if eventhub is given, why user need to provide log analytics parameters?

Reply:
You are correct. I will only validate if upload target is appropriate.